src/PAModelTests/ErrorTests.cs

```diff
@@ -55,5 +58,148 @@ public void OpenMissingSources()
             Assert.IsTrue(errors.HasErrors);
             Assert.IsNull(doc);
         }
+
+        [TestMethod]
```

Comment:
You may want to consider using inline data to make these unit tests data driven
https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-with-dotnet-test#add-more-tests

Reply:
Added!